### PR TITLE
Set accessibility result order by default

### PR DESCRIPTION
**Pull request by mikkoisa**
- Use accessibility order by default if any accessibility settings are selected.

### CODE REVIEW

**Comment by mikkoisa** [comment]
Not sure about the override importance. In these changes accessibility settings will override result orderer initialOrder prop. However, changing home address on address search bar will override accessibility ordering.

File: src/components/ResultOrderer/ResultOrderer.js [hunk 1]
```diff
@@ -37,6 +38,14 @@ const ResultOrderer = ({
   setOrder,
   userLocation,
 }) => {
+  const userSettings = useSelector(state => state.settings);
+  const accessibiliySettingsLength = [
+    userSettings.mobility,
+    userSettings.colorblind,
+    userSettings.hearingAid,
+    userSettings.visuallyImpaired,
+  ].filter(i => (i !== false && i !== null)).length;
```
Comment (Dremars) [inline src/components/ResultOrderer/ResultOrderer.js:47]: I think we should move this to settings utility class as static function hasAccessibilitySettings or accessibilitySettingsActive etc.

**Review by Dremars — CHANGES_REQUESTED** [review]

**Comment by Dremars** [comment]
I think override order should be adress -> accessibility -> most relevant. Accessibility is setting which user selects for general use but address which is set by user for specific action or view.

File: src/components/ResultOrderer/ResultOrderer.js (hunk 1 above)
Comment (mikkoisa) [reply src/components/ResultOrderer/ResultOrderer.js]: Implemented this on latest commits

**Review by Dremars — APPROVED** [review]

**Pull request merged**